src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -29,6 +29,9 @@ namespace UiaOperationAbstraction
 {
     using unique_safearray = wil::unique_any<SAFEARRAY*, decltype(&::SafeArrayDestroy), ::SafeArrayDestroy>;
 
+    template <typename StandinT>
+    using CastFuncType = StandinT (winrt::Microsoft::UI::UIAutomation::AutomationRemoteAnyObject::*)() const;

```

Comment:
I am honestly impressed this all worked lol. Could you add a comment here explaining what this type is and how it's used, since the syntax is super uncommon.

Reply:
LOL, can't agree more.